src/Services/InsuranceService.php

```diff
@@ -74,12 +74,25 @@ public function getOneByPid($id, $type)
     public function search($search, $isAndCondition = true)
     {
         $sql = "SELECT `insurance_data`.*,
-                       `puuid`
+                       `puuid`,
+                       `insureruuid`
                 FROM `insurance_data`
+                LEFT JOIN (
+                    SELECT
+                    CONCAT_WS('-',
+                        HEX(SUBSTRING(`uuid`, 1, 4)),
+                        HEX(SUBSTRING(`uuid`, 5, 2)),
+                        HEX(SUBSTRING(`uuid`, 7, 2)),
+                        HEX(SUBSTRING(`uuid`, 9, 2)),
+                        HEX(SUBSTRING(`uuid`, 11, 6))
+                    ) AS `insureruuid`,
```

Comment:
thanks for the PR @bgandy94!

Have you found this necessary instead of converting the binary to a string later?

Reply:
Great callout. After you said that, I actually found a pre-existing mechanism that does this for us. Makes sense now why `uuid` and `puuid` was working but not the new `insureruuid`.

I updated the PR to remove the complicated query and just add `insureruuid` to the class's `getUuidFields` function